Update of OpenShift task from 2.0.12 to 2.0.13 in Azure DevOps results in "auth data not present" error

Hi.

I now, this is not "true issue" as i already resolve problem, but i think this breaking change should be mentioned in documentation or at least in issue list for other people in future, as i wasted some time to figure out what happened.

Last release 1.6.0 (April 24, 2020) introduced in Azure DevOps OpenShift task "Execute oc command" changes that cause that Azure DevOps release pipeline created using plugin 2.0.12 is failing with following error:

> 2020-04-27T08:58:39.5151322Z ##[section]Starting: Apply Kubernetes manifest
> 2020-04-27T08:58:39.5155091Z ==============================================================================
> 2020-04-27T08:58:39.5155145Z Task         : Execute oc command
> 2020-04-27T08:58:39.5155192Z Description  : OpenShift execute CLI command.
> 2020-04-27T08:58:39.5155223Z Version      : 2.0.13
> 2020-04-27T08:58:39.5155253Z Author       : Red Hat
> 2020-04-27T08:58:39.5155281Z Help         : See [getting started](https://github.com/redhat-developer/openshift-vsts/blob/master/docs/getting-started.md)
> 2020-04-27T08:58:39.5155328Z ==============================================================================
> 2020-04-27T08:58:39.9285370Z [command]/usr/bin/curl -s -L -o /data/myagent/_work/r2/a/.download/openshift-client-linux.tar.gz https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz
> 2020-04-27T08:58:41.1956687Z 
> 2020-04-27T08:58:42.4457140Z ##[error]Endpoint auth data not present: undefined
> 2020-04-27T08:58:42.4501546Z ##[error]Endpoint not present: undefined
> 2020-04-27T08:58:42.4721412Z ##[section]Finishing: Apply Kubernetes manifest

I find out, this is caused by fact that version 2.0.12 generates pipeline task definition that is not compatible with version 2.0.13 and i had to load, modify (any irrelevant change is good enough) and save release pipeline so new version of definition is created and saved.

--- a/README.md
+++ b/README.md
@@ -19,6 +19,9 @@ Below you can find a collection of resources to better undestand how the extensi
 * [Blog on developers.redhat.com](https://developers.redhat.com/blog/2019/12/05/introduction-to-the-red-hat-openshift-deployment-extension-for-microsoft-azure-devops/)
 * [Demo Video](https://www.youtube.com/watch?v=RBwpedmkvow)
 
+## Known Issue
+
+New version 2.0.13 changed all the tasks definitions. This might cause old pipelines, created using plugin 2.0.12 or older, to fail during execution. The issue can be easily solved by load, modify (any irrelevant change is good enough) and save the old pipeline so new definition is created and saved. [#170](https://github.com/redhat-developer/openshift-vsts/issues/170) Thanks to Ludovit Varga.
 
 ## How to contribute
 

